src/regions/ca/info/faq/38_mask.md.fr

```diff
@@ -0,0 +1,15 @@
+## Est ce que je dois porter un masque lorsque je sors dehors?
+
+Le port d’un masque, en dehors de lieux avec directives particulières (hôpitaux, cliniques…) n’est pas recommandé pour les personnes en bonne santé.
+
+En effet, il existe plusieurs risques associés au port du masque :
+- Un faux effet de sécurité, qui réduit votre vigilance à ne pas toucher de multiples surfaces, à ne pas vous laver les mains ou à éviter de vous toucher le visage
+- Un risque de contamination lorsque vous mettez, ajustez ou enlever et jeter votre masque
+
+Il est cependant important de continuer à éviter tout rassemblement, bien vous lavez les mains, touchez les surfaces avec un tissu et garder une distance de 2 mètres avec les autres personnes.
+
+Si vous avez des problèmes de santé, qu’ils soient des problèmes respiratoires aigües (toux, nez qui coule…), chronique ou une condition qui diminue votre système immunitaire, ou si vous prenez soin de quelqu’un qui est infecté, il est cependant conseillé de porter un masque, de bien l’ajuster dès la pose, de le changer régulièrement et de l’enlever par les lanières puis de vous laver les mains.
+
+Voici comment prendre soin de quelqu’un d’infecté au COVID : https://www.canada.ca/fr/sante-publique/services/publications/maladies-et-affections/comment-prendre-soin-personne-atteinte-covid-19-maison-conseils-soignants.html
```

Comment:
Would recommend putting the link as a hyperlink instead of having a full URL in the page

Reply:
Good suggestion! Fixed!